zookeeper-server/src/main/java/org/apache/zookeeper/server/NettyServerCnxn.java

```diff
@@ -528,7 +528,7 @@ private void receiveMessage(ByteBuf message) {
                         ZooKeeperServer zks = this.zkServer;
                         if (zks == null || !zks.isRunning()) {
                             LOG.info("Closing connection to {} because the server is not ready",
```

Comment:
it seem that you are not printing the state

Reply:
Yeah! Good catch, thanks!